azurerm/resource_arm_managed_disk.go

```diff
@@ -173,6 +185,24 @@ func resourceArmManagedDiskCreateUpdate(d *schema.ResourceData, meta interface{}
 		createDisk.DiskProperties.DiskSizeGB = &diskSize
 	}
 
+	if strings.EqualFold(storageAccountType, string(compute.UltraSSDLRS)) {
+		if d.HasChange("disk_iops_read_write") {
+			v := d.Get("disk_iops_read_write")
+			diskIOPS := int64(v.(int))
+			createDisk.DiskProperties.DiskIOPSReadWrite = &diskIOPS
+		}
+
+		if d.HasChange("disk_mbps_read_write") {
+			v := d.Get("disk_mbps_read_write")
+			diskMBps := int32(v.(int))
+			createDisk.DiskProperties.DiskMBpsReadWrite = &diskMBps
+		}
+	} else {
+		if d.HasChange("disk_iops_read_write") || d.HasChange("disk_mbps_read_write") {
+			return fmt.Errorf("Both disk_iops_read_write and disk_mbps_read_write are only available for UltraSSD disks")
```

Comment:
Suggest add a [ERROR] as the prefix of the error message at other returned err msg.

Reply:
Updated code